### PR TITLE
enable planning extension

### PR DESCRIPTION
**Pull request by tomaskikutis**
DEPENDS ON: https://github.com/superdesk/superdesk-client-core/pull/3188
DEPENDS ON: https://github.com/superdesk/superdesk-planning/pull/1383

### CODE REVIEW

File: client/anp/index.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 import photo from './photo';
+import {startApp} from 'superdesk-core/scripts/index';
+import planningExtension from 'superdesk-planning/client/planning-extension/dist/src/extension';
+
+setTimeout(() => {
```
Comment (petrjasek) [inline client/anp/index.ts:5]: why it needs the timeout?

Comment (tomaskikutis) [reply client/anp/index.ts:5]: To work around circular dependency. client-core is importing this via `importApps` and I need to import `startApp` from client core. As we move stuff to extensions, we'll remove `importApps` someday, but until then a timeout is needed.

Comment (petrjasek) [reply client/anp/index.ts:5]: so the import is fine but you can't call that function right away?

Comment (tomaskikutis) [reply client/anp/index.ts:5]: Yep.

**Pull request closed without merging**